editor/assets/primitives.fbx.meta

```diff
@@ -11,15 +11,16 @@
       "displayName": "",
       "id": "17020",
       "name": "sphere.mesh",
-      "ver": "1.1.0",
```

Comment:
Should change meta file?

Reply:
No need, it should be an error in the merge, I have restored it yesterday. I'm checking why it didn't take effect.